src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,27 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    public static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("CompleteOrder method was called. User id: {}", userId);
```

Comment:
Same. Use here debug level and info level you can use in main class, where you invoke this login method. And debug level will be used when it will be necessary.